types/address_test.go

```diff
@@ -320,47 +319,6 @@ func (s *addressTestSuite) TestVerifyAddressFormat() {
 	s.Require().EqualError(err, "address max length is 255, got 256: unknown address")
 }
 
-func (s *addressTestSuite) TestCustomAddressVerifier() {
```

Comment:
Since `addressVerifier` [logic](https://github.com/cosmos/cosmos-sdk/blob/321aed823c882ac893c739c8a4d50905ec702c62/types/address.go#L141-L144) has been removed from `VerifyFormat` and `VerifyAddressFormat` calls `VerifyFormat` I would like to remove `TestCustomAddressVerifier` from codebase.
```
func VerifyAddressFormat(bz []byte) error {
	return address.VerifyFormat(bz)
}
```
```
// VerifyFormat checks validity of address format
func VerifyFormat(bz []byte) error {
	if len(bz) == 0 {
		return sdkerrors.Wrap(sdkerrors.ErrUnknownAddress, "addresses cannot be empty")
	}

	if len(bz) > MaxAddrLen {
		return sdkerrors.Wrapf(sdkerrors.ErrUnknownAddress, "address max length is %d, got %d", MaxAddrLen, len(bz))
	}

	return nil
}
```